bin/docker-entrypoint.sh

```diff
@@ -117,37 +113,28 @@ if [ -z "$(pip list --format=columns | grep "/odoo" | grep -v "/odoo/src")" ]; t
   chown -R odoo: /odoo/*.egg-info
 fi
 
-
 # Wait until postgres is up
 wait_postgres.sh
 
-mkdir -p /data/odoo/{addons,filestore,sessions}
-if [ ! "$(stat -c '%U' /data/odoo)" = "odoo" ]; then
-  chown -R odoo: /data/odoo
-fi
-if [ ! "$(stat -c '%U' /var/log/odoo)" = "odoo" ]; then
-  chown -R odoo: /var/log/odoo
-fi
-
 BASE_CMD=$(basename $1)
 CMD_ARRAY=($*)
 ARGS=(${CMD_ARRAY[@]:1})
 
-if [ "$BASE_CMD" = "odoo" ] || [ "$BASE_CMD" = "odoo.py" ] || ([ "$BASE_CMD" = "gosu" ] && [[ "${ARGS[@]}" =~ "odoo migrate" ]] ); then
+if [ "$BASE_CMD" = "odoo" ] || [ "$BASE_CMD" = "odoo.py" ] || ([ "$BASE_CMD" = "gosu" ] && [[ "${ARGS[@]}" =~ "odoo migrate" ]]); then
```

Comment:
I think the bit about gosu odoo migrate needs rewriting because we are removing gosu:
```suggestion
if [ "$BASE_CMD" = "odoo" ] || [ "$BASE_CMD" = "odoo.py" ] || [ "$BASE_CMD" = "migrate" ]; then
```

Reply:
to check